### PR TITLE
Better accounting of the post revisions when testing the pagination

### PR DESCRIPTION
**Pull request by eri-trabiccolo**
## Description
test fixes.

## How has this been tested?
unit tests

## Types of changes
test fix

## Checklist:
- [x] My code has been tested.
- [x] My code passes all existing automated tests. 
- [x] My code follows the LifterLMS Coding & Documentation Standards. 

@thomasplevy I'm doing this PR against trunk because the dev branch is behind trunk.

### CODE REVIEW

File: tests/unit-tests/server/class-llms-rest-test-lessons.php [hunk 1]
```diff
@@ -348,10 +348,11 @@ public function test_get_items_pagination() {
 
 		wp_set_current_user( $this->user_allowed );
 
-		// create sections
-		$course = $this->factory->course->create_and_get( array( 'sections' => 1, 'lessons' => 25 ) );
+		// Create lessons.
+		$course = $this->factory->course->create_and_get( array( 'sections' => 1, 'lessons' => 25, 'quiz' => 0 ) );
```
Comment (thomasplevy) [inline tests/unit-tests/server/class-llms-rest-test-lessons.php:352]: :+1:

File: tests/unit-tests/server/class-llms-rest-test-lessons.php [hunk 2]
```diff
@@ -348,10 +348,11 @@ public function test_get_items_pagination() {
...
 		$start_lesson_id = $course->get_lessons( 'ids' )[0];
 
+		// When crea
```
Comment (thomasplevy) [inline tests/unit-tests/server/class-llms-rest-test-lessons.php]:
```suggestion
```

**Review by thomasplevy — CHANGES_REQUESTED** [review]

File: tests/unit-tests/server/class-llms-rest-test-lessons.php [hunk 3]
```diff
@@ -352,7 +352,6 @@ public function test_get_items_pagination() {
 		$course = $this->factory->course->create_and_get( array( 'sections' => 1, 'lessons' => 25, 'quiz' => 0 ) );
 		$start_lesson_id = $course->get_lessons( 'ids' )[0];
 
-		// When crea
```
Comment (eri-trabiccolo) [inline tests/unit-tests/server/class-llms-rest-test-lessons.php]: Geez

**Comment by thomasplevy** [comment]
Thanks @eri-trabiccolo <3

**Pull request merged**